nixos/modules/virtualisation/nixos-containers.nix

```diff
@@ -649,6 +649,14 @@ in
               '';
             };
 
+            restartIfChanged = mkOption {
+              type = types.bool;
+              default = true;
+              description = lib.mdDoc ''
+                Whether auto-started containers are restarted on rebuild.
```

Comment:
nitpicking

I think something like "Whether the container should be restarted during a NixOS configuration switch if its definition has changed" is a bit better.

"auto-started" is not relevant here and the change of container setting should be mentioned.

Reply:
SGTM